Proposed/asc056.md

```diff
@@ -0,0 +1,50 @@
+#### **Title**
+Add `informatics` to Autoprotocol instruction schema
+
+#### **Authorship**
+Asuka Ota <asuka.ota@strateos.com>
+
+#### **Motivation**
+In the current form, Autoprotocol is unable to capture scientific intent at the molecular level upon execution of an 
+`instruction`. One of the examples is PCR whereby a mixture of nucleotides and polymerase are mixed in a vial to 
+synthesize new molecules via `thermocycle`. In another case, a user may want to purify a compound of interest from 
+a mixture of compounds in instructions such as `spe` and/or `lcms`. In addition to the physical execution parameters, 
+there should be a set of parameters describing the intended effects of an instruction on the sample composition.
+
+#### **Proposal**
+Instruction currently has `op`, and `data` attributes. We propose adding a new optional attribute, `informatics,` to the 
+`Instruction` to represent the intended effects of an instruction. This may enable future integration of the data 
+specified in `informatics` with vendor LIMS systems. This `informatics` is an optional parameter that can accept a list 
```

Comment:
nit: remove `systems` (LIMS stands for Laboratory Information Management System)